### ISSUE TITLE
`useAliasesForRenames` does not prevent rename in the original file in another project

### ISSUE THREAD

**Issue by esdmr** [microsoft/TypeScript#45734]
# Bug Report

### 🔎 Search Terms

Rename refactor, Named import, Alias for renames, project, tsconfig, folders, directories

### 🕗 Version & Regression Information

This was not fixed in the most recent nightly version 4.5.0-dev.20210905.

- This changed between versions 3.1.8 and 3.2.1
- This changed between nightly versions 3.2.0-dev.20181114 and 3.2.0-dev.20181115

### ⏯ Playground Link

Typescript workbench apparently does not support JSON files, therefore it can not create multiple projects required to reproduce this bug.

Here is the link to [GitHub MWE Repo](https://github.com/esdmr/ts-bug-mwe-rename-named).

### 💻 Code

Make sure that `typescript.preferences.useAliasesForRenames` is enabled (default).

1. Create two directories. One at `lib`, and the other at `src`.
2. In both directories create a `tsconfig.json` with any content. An empty object will do.
3. At `lib/lib.ts` create a file with the following content:
   ```ts
   export const variable = 123;
   ```
4. At `src/main.ts` create a file with the following content:
   ```ts
   import { variable } from '../lib/lib.js';
   ```
5. Close all tabs.
6. Open `lib/lib.ts` and then open `src/main.ts`.
7. Rename `variable` in `src/main.ts`.

### 🙁 Actual behavior

In addition to creating an alias in `src/main.ts` like the following `variable as newName`, the original file `lib/lib.ts` also has its exported variable `variable` renamed to `newName`. This causes Typescript to err that `lib/lib.js has no exported member named variable`.

### 🙂 Expected behavior

If `typescript.preferences.useAliasesForRenames` is enabled, then only create the alias. Do not rename the exported variable as well. This behaviour is currently done when the two files are in the same project.

**Comment by andrewbranch** [comment]
Related: #45659

**Comment by esdmr** [comment]
This is fixed by #48758. :tada:
